app/views/refine/inline/filters/_add_first_condition_button.html.erb

```diff
@@ -6,13 +6,18 @@
     conjunction: "and",
     position: position
   )
+  use_modal ||= false
+  fill_modal ||= false
+  src = use_modal ? index_advanced_refine_inline_criteria_path(criterion.to_params.merge(fill_modal: fill_modal)) : refine_inline_criteria_path(criterion.to_params)
```

Comment:
You'll see this replicated in a couple places. Open to any ideas for more cleanly doing this but I burned quite a bit of time already trying to find something better.